docs/rules/exports-last.md

```diff
@@ -0,0 +1,48 @@
+# exports-last
+
+This rule reports all export declaration which come before any non-export statements.
+
+
+## This will be reported
+
+```JS
+
+const bool = true
+
+export default bool
+
+const str = 'foo'
+
+```
+
+```JS
+
+export const bool = true
+
+const str = 'foo'
+
+```
+
+## This will not be reported
+
+```JS
+export const bool = true
```

Comment:
Add a simple non-exported variable declaration up top, to show that that is allowed